plaidml/op/lib/ops.cc

```diff
@@ -1659,9 +1659,8 @@ Value explicit_padding(const Value& value) {
       IVLOG(2, "Constant padding requested");
 
       auto padval = args[4].as_tensor();
-      I = I - padval;
       O(O_idxs) = I(I_idxs);
-      O = O + padval;
+      O.use_default(padval);
```

Comment:
FYI, this works here because it's an assignment aggregation. But as a warning for elsewhere, `use_default` sets the initial value -- so if it's aggregated with e.g. sum and initialized to neg_inf, in the non-padding region this would just add a bunch of stuff to neg_inf, which isn't going to work. Just an FYI though, this is 100% OK here b/c of the assignment agg_op.